### PR TITLE
Adds activity events

### PR DESCRIPTION
**Pull request by deadlybutter**
Completes message broker service logic and add its it to the events module. Adds new logic, markup and styling for the different box types. 

@DFurnes 

(also pinging @DeeZone since you were interested in what I was doing)

### CODE REVIEW

File: public/js/main.js [hunk 1]
```diff
@@ -160,26 +167,43 @@ $(document).on('ready', function() {
...
+  function handleUserEvent(data) {
+    var container = $('.slide-feed');
+    var children = container.children();
+    if(userBoxIndex >= children.length){
+      userBoxIndex = 0;
+    }
+    var name = data.name || "Another Do-er";
+    switch(data.activity) {
+      case "user_register":
+        updateBox('box--register', '<p><strong>' + name + "</strong> created an account!</p>", container, userBoxIndex);
```
Comment (DeeZone) [inline public/js/main.js]: Consider adding the markup to some kind of template file? It feels odd to see HTML markup in code.

File: services/message_broker.js [hunk 2]
```diff
@@ -5,13 +5,16 @@ module.exports = function(router) {
   var message_broker = {};
 
   router.post('/', function(req, res) {
-    notify("");
+    notify(req.data);
+    res.json(["OKAY", 200]);
```
Comment (DeeZone) [inline services/message_broker.js]: Perfect!

File: public/js/main.js (hunk 1 above)
Comment (deadlybutter) [inline public/js/main.js]: Implement templating for hiding two elements? seems overkill

**Comment by DFurnes** [comment]
Seems good to me! :ship:

**Pull request merged**